pkg/gatewayserver/io/basicstationlns/basicstationlns.go

```diff
@@ -435,6 +431,11 @@ func (s *srv) handleTraffic(c echo.Context) (err error) {
 				logger.WithError(err).Debug("Failed to parse join-request message")
 				return err
 			}
+			// TODO: Remove (https://github.com/lorabasics/basicstation/issues/74)
```

Comment:
This probably has to stay here even after 74 is going to be solved, since it's basically guaranteed that gateway vendors won't upgrade right away.

Reply:
Yep, I'm afraid so indeed.